src/transformers/utils/quantization_config.py

```diff
@@ -167,7 +167,7 @@ class BitsAndBytesConfig(QuantizationConfigMixin):
         bnb_4bit_compute_dtype (`torch.dtype` or str, *optional*, defaults to `torch.float32`):
             This sets the computational type which might be different than the input time. For example, inputs might be
             fp32, but computation can be set to bf16 for speedups.
-        bnb_4bit_quant_type (`str`, {fp4, nf4}, defaults to `fp4`):
+        bnb_4bit_quant_type (`str`, {fp4, nf4}, defaults to `fp4`, *optional*, defaults to `"fp4"`):
```

Comment:
this is a very edgy case
```suggestion
        bnb_4bit_quant_type (`str`,  *optional*, defaults to `"fp4"`):
```

Reply:
Yes in those cases the script does that (I can't add the optional at the right place) but it should be easy to spot in reviews.